UKHomeOffice/sgmr-service
# SPCR-160 Update babel & associated packages

## Description
Updating packages related to Babel

- removed unused/outdated packages
- updated babel packages
- updated eslint packages (babel requires eslint 7.5+, airbnb & other linting packages needed updating to use that)
- turn off some new baseline lint rules that we have not used in the past
- fix some newly discovered errors
- exclude Cypress from linting rules, it will be readded as part of SPCR-94

## To test
- remove package-lock and node-modules
- `npm i`
- `npm start`
- > app should compile
- `npm lint` (for all files)
- > there should be no lint errors


## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
